app/models/trip.rb

```diff
@@ -371,20 +377,20 @@ def return_trip_later_than_outbound_trip
     end
   end
 
-  def compute_run    
+  def compute_run
     return true if run_id || cab || vehicle_id.blank? || provider_id.blank?
 
-    if !pickup_time or !appointment_time 
+    if !pickup_time or !appointment_time
       return true # we'll error out in validation
     end
 
     Trip.transaction do
-      # When the trip is saved, we need to find or create a run for it. This 
-      # will depend on the driver and vehicle.  
-      self.run = Run.where("scheduled_start_time <= ? and scheduled_end_time >= ? and vehicle_id=? and provider_id=?", pickup_time, appointment_time, vehicle_id, provider_id).first
+      # When the trip is saved, we need to find or create a run for it. This
+      # will depend on the driver and vehicle.
+      self.run = Run.where("pickup_time <= ? and scheduled_end_time >= ? and vehicle_id=? and provider_id=?", pickup_time, appointment_time, vehicle_id, provider_id).first
```

Comment:
this doesn't look right. Should restore `scheduled_start_time`.
(Despite the fact that this compute_run function is basically deprecated)